docs/Operators.md

```diff
@@ -6253,6 +6254,98 @@ opset_import {
 </dl>
 
 
+### <a name="TopK"></a><a name="topk">**TopK**</a>
+
+  Retrieve the top-K elements along a specified dimension. Given an input tensor of
+  shape [a_1, a_2, ..., a_n, r] and integer argument k, return two outputs:
+    -Value tensor of shape [a_1, a_2, ..., a_{dim-1}, k, a_{dim+1}, ... a_n] which
+      contains the values of the top k elements along the specified dimension
+    -Index tensor of shape [a_1, a_2, ..., a_{dim-1}, k, a_{dim+1}, ... a_n] which
+     contains the indices of the top k elements (original indices from the input
+     tensor).
+  
+  Given two equivalent values, this operator uses the indices along the dim-
+  ension as a tiebreaker. That is, the element with the lower index will appear
+  first.
+
+#### Versioning
+
+This operator is used if you are using version 1 of the default ONNX operator set until the next BC-breaking change to this operator; e.g., it will be used if your protobuf has:
+
+~~~~
+opset_import {
+  version = 1
+}
+~~~~
+
+#### Attributes
+
+<dl>
+<dt><tt>dim</tt> : int</dt>
```

Comment:
My recommendation would be to name this attribute ``axis``, in line with other ONNX ops such as ``concat``, ``div``, ``flatten``, ``gather`` and others.